ci/ci_steps.sh

```diff
@@ -37,8 +37,11 @@ else
     cp -f "$ENVOY_BUILD_DIR"/envoy/source/exe/envoy ./build_release
     # This script builds a lyft/envoy image and pushes that image on merge to master.
     ./ci/docker_push.sh
-    # This script runs on every PRs release run to test the docker examples.
-    ./ci/verify_examples.sh
+    if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]
```

Comment:
It does, and it already has a guard there. Although you reminded me that we did some work there on every PR only because the images build on docker_push.sh were needed for verification. Now that we only verify on master, we can move those build into the master guard; re: second commit.